packages/cluster-ui/src/store/reducers.ts

```diff
@@ -39,3 +41,16 @@ export const rootReducer = combineReducers<AdminUiState>({
   terminateQuery,
   uiConfig,
 });
+
+export const rootActions = {
+  resetState: createAction(`${DOMAIN_NAME}/RESET_STATE`),
+};
+
+/**
+ * rootReducer consolidates reducers slices and cases for handling global actions related to entire state.
+ **/
+export const rootReducer = createReducer(undefined, builder => {
+  builder
+    .addCase(rootActions.resetState, () => createStore(reducers).getState())
```

Comment:
this is an interesting way to get the initial state. It makes total sense, but not what I expected! Love it.